dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -1591,10 +1592,10 @@ protected void getSnappyJobsStatus(String snappyJobScript, File logFile) {
             }
             inputFile.close();
             for (String str : jobIds) {
-                String command = "while [ \"$(" + snappyJobScript + " status --lead " + leadHost + ":" + LEAD_PORT + " --job-id " + str + " | grep FINISHED | wc -l)\" !=  \"1\" ] ; do sleep 10  ; done";
+                String command = "while [ \"$(" + snappyJobScript + " status --lead " + leadHost + ":" + LEAD_PORT + " --job-id " + str + " | grep status | grep FINISHED | wc -l)\" !=  \"1\" ] ; do sleep 10  ; done";
                 ProcessBuilder pb = new ProcessBuilder("/bin/bash", "-c", command);
```

Comment:
please grep for exact line like status:Finished else Finished could be encountered in multiple rows and the condition will  not match